examples/adv_diff/ex3/example.cpp

```diff
@@ -217,6 +217,8 @@ main(int argc, char* argv[])
         Pointer<BrinkmanAdvDiffSemiImplicitHierarchyIntegrator> bp_adv_diff_hier_integrator = time_integrator;
         bp_adv_diff_hier_integrator->registerBrinkmanAdvDiffBcHelper(brinkman_adv_diff);
 
+        bp_adv_diff_hier_integrator->setBrinkmanTimeIndepedent(q_var, true /* brinkman_time_independent */);
+
```

Comment:
bp_adv_diff_hier_integrator->setBrinkmanVariableTimeIndepedent(q_var, /*time_independent*/ true);

Reply:
We still need to make this correction (etc).